src/js/components/Notification/Notification.js

```diff
@@ -48,36 +93,107 @@ const Notification = ({ message, onClose, id, status, title, toast }) => {
   const { icon: StatusIcon, color } = theme.notification[status];
   const { color: closeIconColor } = theme.notification.close;
 
+  const kind = useMemo(() => {
+    if (toast) return 'toast';
+    if (global) return 'global';
+    return undefined;
+  }, [global, toast]);
+
+  let direction;
+  if (kind && theme.notification[kind].direction)
+    direction = theme.notification[kind].direction;
+  else direction = theme.notification.direction;
+
+  let background;
+  if (kind && theme.notification[status][kind]?.background)
```

Comment:
Will this work to allow turning off the background by setting `background: undefined` in the theme?

Reply:
You mean to explicitly set "toast" background (for example) to undefined and not fallback to other backgrounds?

Reply:
yes, need a way to turn off default behavior. I thought we had an example of this elsewhere, perhaps not though.

Reply:
For example, if we have `theme.notification.container.background: 'blue'` but we don't wan't for only critical toast notifications, we could do `theme.notification.critical.toast.container.background: undefined`.

Reply:
Well, I just skimmed through StyledButtonKind.js and didn't find an example like this. Maybe I'm just dreaming. I'm OK not worrying about it for now. We'll have some time on the stable branch to find any tricky things we don't cover before merging.